truss-chains/truss_chains/definitions.py

```diff
@@ -408,6 +408,7 @@ class ServiceDescriptor(SafeModel):
     specifically with ``StubBase``."""
 
     name: str
+    display_name: str
```

Comment:
Would this data model be a good place to document the backend<->truss discrepancy of naming? Or maybe in `truss/remote` data models?

Reply:
I put the comment under `populate_chainlet_service_predict_urls` since that's where the issue is most prominent.